build/cqs.tsv

```diff
@@ -11,6 +14,7 @@ com.esotericsoftware:minlog	1.3.0	compile
 com.fasterxml.jackson.core:jackson-annotations	2.9.7	compile
 com.fasterxml.jackson.core:jackson-core	2.9.7	compile
 com.fasterxml.jackson.core:jackson-databind	2.9.7	compile
+com.fasterxml.jackson.dataformat:jackson-dataformat-cbor	2.6.7	compile
```

Comment:
we should add a pom entry to make this match the rest of the jackson jar versions, at 2.9.7. Those are defined here: https://github.com/locationtech/geomesa/blob/master/pom.xml#L1723